### PR TITLE
Change RMSPE to RMSRE

### PR DESCRIPTION
**Pull request by cucuandreea**

### CODE REVIEW

File: public/js/dom.js [hunk 1]
```diff
@@ -389,29 +389,50 @@ export function displaySimulationResults_simpleMode(
...
-        return 'over-15'
-    } else if (noiseValue >= 10) {
-        return 'over-10'
-    } else if (noiseValue >= 5) {
-        return 'over-5'
-    } else if (noiseValue >= 1) {
-        return 'over-1'
-    } else {
-        return 'under-1'
+function getNoiseBadgeType(noiseValue, percentage) {
```
Comment (maudnals) [inline public/js/dom.js:392]: Change request: Rename to `isPercentage` to stick to conventions for booleans

File: public/js/config.js [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const MODES = {
     },
 }
 
-export const RMSPE_THRESHOLD = 5
+export const RMSRE_THRESHOLD = 5
 
 export const CONTRIBUTION_BUDGET = 65536
```
Comment (maudnals) [inline public/js/config.js:33]: Change request: Bump the version number

**Review by maudnals — CHANGES_REQUESTED** [review]
LGTM % comments. 
Thank you for making the update!

File: public/js/dom.js (hunk 1 above)
Comment (cucuandreea) [reply public/js/dom.js]: Done

File: public/js/config.js (hunk 2 above)
Comment (cucuandreea) [reply public/js/config.js:33]: Done

**Review by cucuandreea — COMMENTED** [review]
Applied requested changes

**Pull request merged**